docs/faq.rst

```diff
@@ -0,0 +1,126 @@
+        - `Fairness and Abstraction in Sociotechnical Systems <https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3265913>`_ (Selbst et al., 2018)
+        - `A Survey on Bias and Fairness in Machine Learning <https://arxiv.org/abs/1908.09635>`_ (Mehrabi et al. 2019)
+        - `Algorithmic Fairness from a Non-ideal Perspective <http://zacklipton.com/media/papers/fairness-non-ideal-fazelpour-lipton-2020.pdf>`_ (Fazelpour & Lipton, 2020)
+
+    Why not just ignore the sensitive features?
+        Because ignoring the sensitive feature usually doesn't work.
+
+        To elaborate, information is often redundantly encoded in datasets, and machine learning
+        algorithms *will* ferret out these links (it is what they are designed to do).
+        For example, in the US, the ZIP code where a person lives is well correlated with their
+        race.
+        Even if the model is not provided with race as a feature, the model will pick up on it
+        implicitly via the ZIP code (and other features).
+        Worse, without having the race available in the dataset, it is not possible to determine
+        whether any use of ZIP code was as a proxy for race, or for some other reason.
+
+    The model is unfair because the data are biased. Isn't it better to get better data?
+        Yes, but gathering unbiased data might not always be practical.
+        Fairlearn and similar packages offer techniques to redistribute harms between subgroups
+        in the data.
+
+    Won't making a model fairer reduce its accuracy?
+        Any technique which reduces disparity in some metric is most likely to impact other metrics.
+        However, we have found empirically that it is usually possible to obtain substantial reductions
+        in disparity (as measured by some suitable metric) at minimal cost in accuracy.
+        Indeed, for certain subgroups in the data, accuracy may be improved dramatically.
+
+.. topic:: Fairlearn Questions
+
+    Can Fairlearn make my model fair?
+       Naive use of Fairlearn is not going to result in fairer models.
+       Indeed, when used incorrectly, Fairlearn could produce model which is *less* fair.
+
+       What Fairlearn can do is generate a model which has lower disparity as measured by
+       a user-specified metric.
+       Whether this makes the model fairer is outside the remit of Fairlearn - or indeed of
+       machine learning in general.
+       To determine whether the new model is fairer, the whole system of which it is a part
+       needs to be considered in terms of the societal context in which it is used.
+
+    What sort of fairness related harms can Fairlearn address?
+        Fairlean is focussed on group fairness.
+        That is, minimising the difference in some metric evalulated on
+        specified subgroups in the data.
+        This is certainly not the only way of looking at fairness, though.
```

Comment:
Cliffhanger :-) Let's just say there's also individual fairness and counterfactual fairness?

Reply:
Agreed.